Aptoide/appcoins-wallet-android
# [MOB-880] Notification improvements

**What does this PR do?**
Adds language support for the walletHistory endpoint
Improves and fixes the following:
- The time when the perk notification appears to the user
- Notification value rounding to match the one in transaction list

**Database changed?**

No

**Where should the reviewer start?**

IabPresenter.kt
TopUpActivityPresenter.kt
PerkBonusService.kt

**How should this be manually tested?**

Make a purchase on trivial drive and check if the notification appears and check if the value matches the on the transaction list

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/MOB-880

**Code Review Checklist**
